glean-core/src/metrics/mod.rs

```diff
@@ -336,6 +343,9 @@ impl Metric {
             Metric::Object(s) => {
                 serde_json::from_str(s).expect("object storage should have been json")
             }
+            Metric::DualLabeledCounter(s) => {
+                json!(s)
```

Comment:
See the other comment, we probably don't need this.
But apart from that, this wouldn't even work. We cannot serialize a `HashMap<(String, String), ...>` to JSON, as all JSON object keys have to be strings.

Again, we need to handle serialization in some other place anyway.